index.html

```diff
@@ -66,5 +72,236 @@ <h3>
+                <li class="box2-li">Css</li>
+                <li class="box2-li">JavaScript</li>
+              </ul>
+              <button class="big-btn">See project</button>
+            </div>
+          </block>
+        </cards>
+      </works>
+    </div>
+
+    <!-- about myself -->
+
+    <About class="about">
+      <section class="section1">
+        <h1 class="primary-text">About Myself</h1>
+        <p class="paragraph about-para">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+        <a href="#" class="connect about-connect">Let's connect</a>
+        <ul class="social">
+          <li>
+            <a href="https://twitter.com/ankit26k">
+              <img class="icon" src="/images/twitter.png" alt="twitter"
+            /></a>
+          </li>
+
+          <li>
+            <a href="https://www.linkedin.com/in/ankit26k/">
+              <img class="icon" src="/images/Linkedin.png" alt="linkedin"
+            /></a>
+          </li>
+
+          <li>
+            <a href="https://twitter.com/ankit26k">
+              <img class="icon meta" src="/images/Vector1.png" alt="meta"
+            /></a>
+          </li>
+
+          <li>
+            <a href="https://github.com/ankitt26">
+              <img class="icon" src="/images/vector2.png" alt="github"
+            /></a>
+          </li>
+
+          <li>
+            <a href="https://www.instagram.com/ankit16t/">
+              <img class="icon" src="/images/vector3.png" alt="twofin"
+            /></a>
+          </li>
+        </ul>
+        <button class="about-btn">Get my resume</button>
+      </section>
+
+      <!-- section 2 -->
+
+      <section>
+        <ul class="section2-ul">
+          <li class="section2-li">
+            <h2 class="section2-h2">Language</h2>
+            <i class="bi bi-chevron-down icon2"></i>
+          </li>
+
+          <div class="languages">
+            <div class="lan">
+              <img class="lan-img" src="/images/javascript.png" alt="" />
+              <h3 class="section2-h3">JavaScript</h3>
+            </div>
+            <div class="lan">
+              <img class="lan-img" src="/images/HTML.png" alt="" />
+              <h3 class="section2-h3">HTML</h3>
+            </div>
+            <div class="lan">
+              <img class="lan-img" src="/images/css.png" alt="" />
+              <h3 class="section2-h3">CSS</h3>
+            </div>
+          </div>
+
+          <li class="section2-li hr">
+            <h2 class="section2-h2">Language</h2>
+            <i class="bi bi-caret-right-fill icon2"></i>
+          </li>
+
+          <li class="section2-li hr">
+            <h2 class="section2-h2">Language</h2>
+            <i class="bi bi-caret-right-fill icon2"></i>
+          </li>
+        </ul>
```

Comment:
Please make sure that you use the same titles as in figma template as the main goal from this project is the be able to replicate the same design as much as possible 

![temp1 languages skills student](https://user-images.githubusercontent.com/59707859/227724482-7115c608-6e14-4fe3-91bd-02a8bfccd102.PNG)
![temp1 languages skills](https://user-images.githubusercontent.com/59707859/227724483-3129ba59-e429-41bc-bb0e-4a95381ce1dc.PNG)